tomviz/ModuleManager.cxx

```diff
@@ -345,6 +357,9 @@ bool ModuleManager::deserialize(const pugi::xml_node& ns, const QDir& stateDir)
   int numViews = 0, numLayouts = 0;
   for (pugi::xml_node node = ns.child("View"); node; node = node.next_sibling("View"))
   {
+    pugi::xml_node agnode = node.child("AxesGrid");
+    pugi::xml_node agproxyNode = agnode.child("Proxy");
```

Comment:
As above, and Proxy should be capitalized, i.e. axesGridProxyNode, axesGridNode, etc.